### PR TITLE
Add vscode launch.json for debugging within Vscode

### PR DESCRIPTION
**Pull request by Sensiblemnd**
debug

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
```
Comment (Sensiblemnd) [inline .vscode/launch.json:3]: comment?

**Comment by Sensiblemnd** [comment]
Should I abandon??

**Comment by AykutSarac** [comment]
Hello, can you share the pros and use-case of this?

**Comment by Sensiblemnd** [comment]
Pros and Cons??

Instead of using chrome dev tools you can debug directly inside VScode.
Add breakpoint inspect var etc

Otherwise use chome devtools and switch back and forth from Chrome to Vscode

**Comment by Sensiblemnd** [comment]
See screenshot

**Comment by Sensiblemnd** [comment]
Abandon?

**Pull request closed without merging**